.env.example

```diff
@@ -1,14 +1,26 @@
 # BOT
 SMF_BOT_BACKEND_URL="http://localhost:5555"
 SMF_BOT_DRIP_AMOUNT=10
+
+# Log in to the bots matrix account. Click on the name in the top left corner, then "Settings".
```

Comment:
It should be told that those instructions you mention are for riot.im specifically